src/Core/Abstractions/IStateService.cs

```diff
@@ -128,8 +128,8 @@ public interface IStateService
         Task SetEventCollectionAsync(List<EventData> value);
         Task<Dictionary<string, FolderData>> GetEncryptedFoldersAsync(string userId = null);
         Task SetEncryptedFoldersAsync(Dictionary<string, FolderData> value, string userId = null);
-        Task<Dictionary<string, PolicyData>> GetEncryptedPoliciesAsync(string userId = null);
-        Task SetEncryptedPoliciesAsync(Dictionary<string, PolicyData> value, string userId = null);
+        Task<Dictionary<string, PolicyData>> GetPoliciesAsync(string userId = null);
```

Comment:
Are these truly no longer encrypted?  If so is there existing data that requires migration?